### ISSUE TITLE
every 2nd column removed

### ISSUE THREAD

**Issue by planetClaire** [derhuerst/svg-radar-chart#4]
Thank you for the great lightweight script which I'm using successfully on 2 charts.  The first has 5 columns & works beautifully.  For the 2nd I had to comment out the code below:
`	columns.reduce((all, column) => {
		all[column.key] = column
		return all
	}, columns)`

For some reason it is transforming data with this set of 18 columns:
`{1: "Collaboration & Teamwork", 2: "Trustworthiness", 3: "Empathy & Respect", 4: "Relationship-centred Care", 5: "Effective Communication", 6: "Application of Expertise", 7: "Problem-solving", 8: "Managing Workflow", 9: "Continual Learning", 10: "Commitment", 11: "Diligence", 12: "Sustainable Engagement", 13: "Motivation", 14: "Resilience", 15: "Adaptability", 16: "Emotional Competence", 17: "Reflective Self-evaluation", 18: "Self-confidence & Identity"}`

into a VirtualNode with every 2nd column replaced with a copy of its predecessor, which then results in an SVG chart with every 2nd column missing.  See screenshots below.  Weird reduce behaviour, or something I'm doing wrong?

![image](https://user-images.githubusercontent.com/379390/30625855-69dcaee0-9df8-11e7-81ab-ca8cb097a37e.png)
![image](https://user-images.githubusercontent.com/379390/30625873-9b684a8c-9df8-11e7-86ea-dcd88505df81.png)

**Comment by derhuerst** [comment]
Thanks for the detailed report. What do the parameters look like you pass into `radar()`?

**Comment by planetClaire** [comment]
Here it is...

```
            const columns = this.props.Ratings.reduce((prev, curr) => {
                prev[curr.Id] = curr.Title;
                return prev;
            }, {});
            console.log(columns);
            const data = this.props.Ratings.reduce((prev, curr) => {
                prev[curr.Id] = curr.Rating / 100;
                return prev;
            }, { class: 'user' });
            console.log(data);
            const options = {
                captionProps: () => ({
                    textAnchor: 'middle', fontSize: 2
                })
            };
            let chart = radar(columns, [data], options);
```

![image](https://user-images.githubusercontent.com/379390/30670211-0176d85c-9e94-11e7-9096-09dfcc7951fc.png)

**Comment by derhuerst** [comment]
Ideed there seems to be a bug. I can confirm it on [the website](https://derhuerst.github.io/svg-radar-chart/) with the following values:

```js
{
	1: 'Price',
	2: 'Usefulness',
	3: 'Design',
	4: 'Battery Capacity',
	5: 'Camera Quality'
}
```

```js
[
	{
		// iphone
		color: '#edc951',
		1: 1,
		2: .7,
		3: 1,
		4: .9,
		5: .9
	}
]
```

**Comment by planetClaire** [comment]
and yet not for:

```
{0: "Effective Relationships", 1: "Veterinary Capabilities", 2: "Professional Commitment", 3: "Psychological Resources", 4: "Self-awareness"}

[{0:0.72,1:0.5,2:0.76,3:0.92,4:0.66}]
```

**Comment by planetClaire** [comment]
starting your example at index 0 fixes it...

**Comment by derhuerst** [comment]
I fixed numeric indexes in a recent commit but haven't pushed & released yet.

**Comment by derhuerst** [comment]
Should be fixed as of `svg-radar-chart@0.1.1`.
